docs/tutorials/tabular_prediction/tabular-multimodal-text-others.md

```diff
@@ -257,12 +257,14 @@ from autogluon.text.text_prediction.text_prediction import ag_text_prediction_pa
 from autogluon.tabular.task.tabular_prediction.hyperparameter_configs import get_hyperparameter_config
 import copy
 
-text_nn_params = ag_text_prediction_params.create('default_no_hpo')
-text_nn_params['models']['BertForTextPredictionBasic']['search_space']['model.backbone.name'] = 'google_electra_base'
+text_nn_params = ag_text_prediction_params.create('default_electra_base_no_hpo')
```

Comment:
Can we think of any shorter names than `'default_electra_base_no_hpo'`, seems a bit long, but perhaps all parts of the name are necessary...

Reply:
Let me think about it in a later PR.